Dockerfile

```diff
@@ -1,14 +0,0 @@
-FROM ocaml/opam:ubuntu-16.04_ocaml-4.04.2
```

Comment:
Maybe we should keep this in utils and add a job that builds `xs-opam:devel` with `xs-opam:latest + upstream + the tools and libraries installed here`

Reply:
Sounds good, I'll remove the commit for now, and leave this change for another PR